main/src/io/github/iltotore/iron/constraint/collection.scala

```diff
@@ -41,6 +41,11 @@ object collection:
    */
   type Empty = Length[StrictEqual[0]] DescribedAs "Should be empty"
 
+  /**
+   * Tests exact length. Supports [[Iterable]] and [[String]] by default.
+   */
+  type FixedLength[V <: Int] = Length[StrictEqual[V]] DescribedAs "Should have an exact length of" + V
```

Comment:
I would expect the `FixedLength` constraint to be defined before `Empty` and make `Empty` an alias of `FixedLength`